### PR TITLE
Updates for constitutional delegates

### PR DESCRIPTION
**Pull request by adamgoth**
In this PR:
- [x] Delegates Page: Recognized Delegate Title becomes 'Constitional Delegates'
- [x] Delegates Page: Status Filter Recognized Delegate becomes 'Constitional Delegates'
- [x] Delegates Page: Remove tag filter element.
- [x] Delegates Page: System Info Panel change Recognized Delegates -> Constitutional Delegates.
- [x] Delegates Page: Update Delegate FAQs panel.
- [x] Main Page: Governance Stats, Recognized Delegates -> Constitutional Delegates
- [x] Delegate Card: Remove tags
- [x] remove 'meet the delegates' section altogether
- [x] repurpose 'top recognized delegates' to reflect 'top constitutional voter committees' (BLOCKED until we know the CVC names)
- [x] change all remaining mentions of 'recognized delegates' into 'constitutional delegates'
- [x] There is one link to https://manual.makerdao.com/delegation/delegate-expiration on the landing page which should be swapped out by https://manual.makerdao.com/delegation/overview/what-is-delegation

### CODE REVIEW

**Review by rafinskipg — APPROVED** [review]

**Review by rafinskipg — APPROVED** [review]

**Comment by b-pmcg** [comment]
LGTM, sad to see delegate tags go

File: modules/delegates/components/DelegatePicture.tsx [hunk 1]
```diff
@@ -76,13 +76,15 @@ export function DelegatePicture({
...
-            {delegate.status === DelegateStatusEnum.recognized ? delegate.name : 'Shadow Delegate'}
+            {delegate.status === DelegateStatusEnum.constitutional ? delegate.name : 'Shadow Delegate'}
           </Text>
           <Text as="p" sx={{ fontSize: 2, mt: 1 }}>
             <Address address={delegate.voteDelegateAddress} />
           </Text>
           <Text as="p" sx={{ fontSize: 2 }}>
-            {delegate.status === DelegateStatusEnum.recognized ? 'Recognized Delegate' : 'Shadow Delegate'}
+            {delegate.status === DelegateStatusEnum.constitutional
+              ? 'Recognized Delegate'
```
Comment (tyler17) [inline modules/delegates/components/DelegatePicture.tsx:86]:
```suggestion
              ? 'Constitutional Delegate'
```

**Review by tyler17 — DISMISSED** [review]
looks good, just commented one edit

**Review by rafinskipg — APPROVED** [review]

**Review by rafinskipg — DISMISSED** [review]

**Review by tyler17 — DISMISSED** [review]

**Review by rafinskipg — DISMISSED** [review]

File: modules/delegates/components/DelegateOverviewCard.tsx [hunk 2]
```diff
@@ -201,9 +200,7 @@ export function DelegateOverviewCard({ delegate }: PropTypes): React.ReactElemen
               sx={{
                 mb: delegate.tags && delegate.tags.length > 0 ? 1 : 0
               }}
-            >
-              <DelegateTags tags={delegate.tags.slice(0, 3)} />
-            </Box>
+            ></Box>
```
Comment (torhector2) [inline modules/delegates/components/DelegateOverviewCard.tsx:203]: Can we remove this Box? it's empty.

**Review by torhector2 — APPROVED** [review]

**Review by b-pmcg — APPROVED** [review]
LGTM!

**Pull request merged**